templates/compose/azimutt.yaml

```diff
@@ -0,0 +1,36 @@
+# documentation: https://docs.azimutt.app/
+# slogan: Next-Gen ERD: Design, Explore, Document and Analyze your database.
+# tags: erd, entity-relationship diagram, database tool, database schema, diagram
+# icon: svgs/azimutt.svg
+# port: 4000
+
+services:
+  database:
+    image: postgres
+    restart: always
+    container_name: azimutt-db
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=$SERVICE_PASSWORD_POSTGRESQL
+      - POSTGRES_USER=$SERVICE_USER_POSTGRESQL
+      - POSTGRES_DB=azimutt_dev
```

Comment:
If you choose the name, maybe `azimutt_app` is better than `azimutt_dev` (that I use for dev env ^^)